homeassistant/components/zwave_js/api.py

```diff
@@ -436,6 +442,76 @@ def filename_is_present_if_logging_to_file(obj: dict) -> dict:
     return obj
 
 
+@websocket_api.require_admin  # type: ignore
+@websocket_api.async_response
+@websocket_api.websocket_command(
+    {
+        vol.Required(TYPE): "zwave_js/start_listening_logs",
+        vol.Required(ENTRY_ID): str,
+    }
+)
+async def websocket_start_listening_logs(
+    hass: HomeAssistant, connection: ActiveConnection, msg: dict
+) -> None:
+    """Start listening to logging messages from the server."""
+    entry_id = msg[ENTRY_ID]
+    client = hass.data[DOMAIN][entry_id][DATA_CLIENT]
+    driver = client.driver
+
+    @callback
+    def async_cleanup() -> None:
+        """Remove signal listeners."""
+        hass.async_create_task(driver.async_stop_listening_logs())
+        for unsub in unsubs:
+            unsub()
+
+    @callback
+    def forward_event(event: dict) -> None:
+        log_msg: LogMessage = event["log_message"]
+        connection.send_message(
+            websocket_api.event_message(
+                msg[ID],
+                {
+                    "timestamp": log_msg.timestamp,
+                    "level": log_msg.level,
+                    "primary_tags": log_msg.primary_tags,
+                    "formatted_message": log_msg.formatted_message,
+                },
+            )
+        )
+
+    connection.subscriptions[msg["id"]] = async_cleanup
+    unsubs = [
+        driver.on("logging", forward_event),
+        async_dispatcher_connect(
+            hass, f"{entry_id}_stop_listening_logs", async_cleanup
+        ),
+    ]
+
+    await driver.async_start_listening_logs()
+    connection.send_result(msg[ID])
+
+
+@websocket_api.require_admin  # type: ignore
+@websocket_api.async_response
+@websocket_api.websocket_command(
+    {
+        vol.Required(TYPE): "zwave_js/stop_listening_logs",
```

Comment:
There is no need for this command, it will be done by the cleanup task of the subscription above.